### PR TITLE
Set CPU affinity for CVMFS processes [v2]

### PR DESCRIPTION
**Pull request by jblomer**
Replaces #3072

### CODE REVIEW

**Comment by jblomer** [comment]
@cernvm-bot cloudtest

**Comment by HereThereBeDragons** [comment]
do you want a small test for this? 

```
You can also retrieve the CPU affinity of an existing task:
    taskset -p pid 
```

**Comment by jblomer** [comment]
@cernvm-bot cloudtest

**Comment by HereThereBeDragons** [comment]
i personally would probably check for setting more than one cpu

**Comment by jblomer** [comment]
@cernvm-bot cloudtest

**Review by vvolkl — APPROVED** [review]

File: test/src/101-cpuaffinity/main [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+cvmfs_run_test() {
+  logfile=$1
+
+  if running_on_osx; then
+    echo "Skipping test on macOS"
+    return 0
+  fi
+
+  echo "*** mount cernvm-prod.cern.ch normally"
```
Comment (HereThereBeDragons) [inline test/src/101-cpuaffinity/main]: maybe change this to `mount <repo> - cpu affinity 0,1`?

**Review by HereThereBeDragons — APPROVED** [review]
just one small improvement, but can also be merged like that

**Pull request merged**